### ISSUE TITLE
Stuck in "Your new encryption password" window

### ISSUE THREAD

**Issue by Nebulean** [Laverna/laverna#735]
Hello,

I changed the option that is by default at 10'000, I changed it to 100'000, just to try.
When I saved, it asked me to enter a new encryption password. I am stuck in that window, and I can't do anything.
<img width="486" alt="screenshot" src="https://cloud.githubusercontent.com/assets/9269185/23738716/f6829ab4-049a-11e7-9da4-ee850e9a1f74.png">

Where are Laverna's data folder ? (to reset)

I am on macOS.

Thanks.

**Comment by RebelCoderRU** [comment]
Looking at the dates.... last build was almost a year ago - Jun 30, 2016. Such a good app left to dry to be used actively. I still use it on my Linux and Android, but it seems to be dead now =(

**Comment by Nebulean** [comment]
Maybe that one day someone will maintain this app, that's why open source exists. ^_^'

**Comment by blacksky86** [comment]
Yeah, I was looking forward to an open source alternative to Evernote - this one fit the description but it's now dead. Weird since it has so many stars.

**Comment by Neats29** [comment]
Same here. but the latest commit was on Mar 1, 2017... so maybe it's not dead?

**Comment by geoff-maddock** [comment]
Came here to look into this bug.   Alas doesn't look very active.   Is there an active fork or similar project out there?

**Comment by ghost** [comment]
I have the same issue: I backed everything up, but when I try to load the settings/notes it prompts me for my  'new encryption password'. I never reset my password so not sure why this would happen. I have a ton of notes that are inaccessible now until this can be sorted out... Please help!

**Comment by funilrys** [comment]
Hello there,
Please report to https://github.com/Laverna/laverna/issues/971#issuecomment-411423965 which explain the state of this project.

For the future of Laverna, please report to https://github.com/daed/laverna .

Have a nice day/night,
Cheers,
Nissar
